docs/src/environments/_index.md

```diff
@@ -186,3 +186,72 @@ Staging
   Developer2
     Ticket-395
 ```
+
+## Paused environments
+
+Development environments are often used for a limited time and then abandoned.
+To prevent unnecessary consumption of resources,
+Platform.sh automatically pauses development environments where code hasn't been pushed in 14 days.
+
+You can also pause an environment manually at any time.
+
+Currently, you can only pause and resume environments through the [Console](../administration/web/_index.md)
+and [Platform.sh API](https://api.platform.sh/).
+In the future, you'll be able to do so in the [CLI](../administration/cli/_index.md) too.
+
+### Pause an environment
+
+To pause an environment, follow these steps:
+
+{{< codetabs >}}
+
++++
+title=In the Console
++++
+
+1. Navigate to your project and click {{< icon settings >}} **Settings**.
+2. Under **Environments**, select the environment you want to pause.
+3. In the **Status** tab, click **Pause environment**.
+
+<--->
++++
+title=Using the API
++++
+
+Run the following command:
+
+```bash
+platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/pause -X POST -p {{< variable "PROJECT_ID" >}}
```

Comment:
@AnouckColson wasn't there a PR recently that addressed that we want to use expanded commands (`project:curl` vs `p:curl`) preferably? What's our standard?